docs/source/traitsui_user_manual/custom_view.rst

```diff
@@ -256,66 +256,151 @@ Other View Attributes
-|          |                     |not used by Traits, but can be used by a     |
-|          |                     |custom help handler.                         |
-|          |                     |                                             |
-|          |                     |.. index:: id attribute; View                |
-+----------+---------------------+---------------------------------------------+
-|Unique    |* **id**             |The **id** attribute is used as a key to save|
-|identifier|                     |user preferences about a view, such as       |
-|          |                     |customized size and position, so that they   |
-|          |                     |are restored the next time the view is       |
-|          |                     |opened. The value of **id** must be unique   |
-|          |                     |across all Traits-based applications on a    |
-|          |                     |system. If no value is specified, no user    |
-|          |                     |preferences are saved for the view.          |
-+----------+---------------------+---------------------------------------------+
-
- 
+
+Window display
+   These attributes control the visual properties of the window itself,
+   regardless of its content.
+
+   dock: {'fixed', 'horizontal', 'vertical', 'tabbed'}
+      The default docking style to use for sub-groups of the view. The following
+      values are possible:
+
+      * 'fixed': No rearrangement of sub-groups is allowed.
+      * 'horizontal': Moveable elements have a visual "handle" to the left by
+        which the element can be dragged.
+      * 'vertical': Moveable elements have a visual "handle" above them by
+        which the element can be dragged.
+      * 'tabbed': Moveable elements appear as tabbed pages, which can be
+        arranged within the window or "stacked" so that only one appears at
+        at a time.
+   height: int or float
+      Requested height for the view window, as an (integer) number of pixels, or
+      as a (floating point) fraction of the screen height.
+   icon: str
+      The name of the icon to display in the dialog window title bar.
+   image: Image
+      The image to display on notebook tabs.
+   item_theme:
+      The default theme to use for a contained item.
+   label_theme:
+      The default theme to use for a contained item's label
+   resizable: bool
+      Can the user resize the window?
+   scrollable: bool
+      Can the user scroll the view? If set to True, window-level scroll bars
+      appear whenever the window is too small to show all of its contents at
+      one time. If set to False, the window does not scroll, but individual
+      widgets might still contain scroll bars.
+   statusbar:
+      Status bar items to add to the view's status bar. The value can be:
+
+      - **None**: No status bar for the view (the default).
+      - string: Same as [ StatusItem( name = string ) ].
```

Comment:
Probably want to quote `[StatusItem(name=string)]` so that it appears as code in final doc, and remove idiosyncratic code style here and below